doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -1,181 +1,196 @@
+When using the ROS interface, the status of Servo is available on the topic ``/servo_node/status``, see definition `ServoStatus <https://github.com/ros-planning/moveit_msgs/blob/ros2/msg/ServoStatus.msg>`_.
 
-Requirements for incoming command messages, regardless of input device are:
-    1. **TwistStamped and JointJog:** need a timestamp in the header that is updated when the message is published
-    2. **JointJog:** must have valid joint names in the :code:`joint_names` field that correspond with the commands given in the :code:`displacements` or :code:`velocities` fields
-    3. **(Optional) TwistStamped:** can provide an arbitrary :code:`frame_id` in the header that the twist will be applied to. If empty, the default from the configs is used
+Launch ROS interface demo: ``ros2 launch moveit_servo demo_ros_api.launch.py``.
 
-Servo Configs
-^^^^^^^^^^^^^
+Once the demo is running, the robot can be teleoperated through the keyboard.
 
-The `demo config file <https://github.com/ros-planning/moveit2/blob/main/moveit_ros/moveit_servo/config/panda_simulated_config.yaml>`_ shows the parameters needed for MoveIt Servo and is well documented.
+Launch the keyboard demo: ``ros2 run moveit_servo servo_keyboard_input``.
 
-Start with the parameters from the demo file, but some must be changed for your specific setup:
-    1. :code:`robot_link_command_frame`: Update this to be a valid frame in your robot, recommended as the planning frame or EEF frame
-    2. :code:`command_in_type`: Set to "unitless" if your input comes from a joystick, "speed_units" if the input will be in meters/second or radians/second
-    3. :code:`command_out_topic`: Change this to be the input topic of your controller
-    4. :code:`command_out_type`: Change this based on the type of message your controller needs
-    5. :code:`publish_joint_positions` and :code:`publish_joint_velocities`: Change these based on what your controller needs. Note if :code:`command_out_type == std_msgs/Float64MultiArray`, only one of these can be True
-    6. :code:`joint_topic`: Change this to be the joint_state topic for your arm, usually :code:`/joint_states`
-    7. :code:`move_group_name`: Change this to be the name of your move group, as defined in your SRDF
-    8. :code:`planning_frame`: This should be the planning frame of your group
+An example of using the pose commands in the context of servoing to open a door can be seen in this `example <https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/examples/realtime_servo/src/pose_tracking_tutorial.cpp>`_.
```

Comment:
I might suggest creating a new subsection here called e.g. "Application Examples" that shows this door opening demo, and later can show some of the other cool servo tutorials planned.

Reply:
Maybe later one we have more application demos ?
Right now it is just the door opening pose demo.